### PR TITLE
0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json

### PR DESCRIPTION
**Pull request by yonathn**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- []  Add a new file (or update existing file) into the `set-details` or `set-details-polygon` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [] If adding brand new token, make sure to add it to the `set.tokenlist.json` or `set.polygon.tokenlist.json` as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

[0xc31835348c91eefe713fcb3bd8ceabf2d1ee5e9a9e11fc4636a4b87ce914154469cd9419d37136f16a83849ee5d354c194c20fc825da901564d51e2a00dcb9b01c]

### CODE REVIEW

**Comment by yonathn** [comment]
whats up with this

File: set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Orticalia Fund",
+    "setSymbol": "ORTIC",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/yonathn/uniswap-tokenlist/1777437c02fce18034267adb97e26194db60952a/assets/tokens/Untitled_design_1_80.png",
```
Comment (justinkchen) [inline set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json:7]: these should not be referencing locations from your own local branch. all assets need to be from the main branch...see other examples for reference, they are from the SetProtocol org and `main` branch

**Comment by yonathn** [comment]
Got you thx

File: set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json (hunk 1 above)
Comment (yonathn) [reply set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json]: Hello, Justin. Thanks for the reply. I just changed it now. what do you think? Do you think it's good now?

Comment (justinkchen) [reply set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json]: No, these are still not referencing real files that will be in this repo,please follow the examples from other tokens, the url should look the same and also the actual image files need to be uploaded as part of the PR too

Comment (yonathn) [reply set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json]: got you thanks i am on it

Comment (yonathn) [reply set-details/0xaa941FeDa812cF478c576F3A82610fB81029C3c4.setdetails.json]: > No, these are still not referencing real files that will be in this repo,please follow the examples from other tokens, the url should look the same and also the actual image files need to be uploaded as part of the PR too

is it good now I add the icons and I changed the URL by the way thank you so much for your time man

**Pull request merged**